.gitignore

```diff
@@ -0,0 +1,7 @@
+node_modules
+randomHexColor.js
+draft.js
+colorNameToHex.js
+yarn.error.lock
+
+
```

Comment:
I would suggest removing empty lines from the end of files - just as a general good practice.